### PR TITLE
FEATURE: Adding metrics - num_get_vote_accounts_voting, num_live_validator_histories

### PR DESCRIPTION
**Pull request by coachchucksol**
Added two new metrics to the `emit_validator_history_metrics` function:

- `num_live_validator_histories`: The number of validator history accounts that have "live" vote accounts with the owner of said vote account being the vote program
- `num_get_vote_accounts_voting`: The number of vote accounts actually voting. More specifically, accounts that have voted in the current epoch retrieved from `get_vote_accounts_with_retry`.

### CODE REVIEW

File: keepers/validator-keeper/src/lib.rs [hunk 1]
```diff
@@ -216,6 +244,7 @@ pub async fn emit_validator_history_metrics(
             get_vote_accounts_count,
             i64
         ),
+        ("num_get_vote_accounts_live_validators", get_vote_accounts_live_validators, i64),
```
Comment (ebatsell) [inline keepers/validator-keeper/src/lib.rs:247]: these names are a bit confusing, maybe 
`num_get_vote_accounts_voting`
`num_live_validator_histories`
open to other suggestions

File: keepers/validator-keeper/src/lib.rs [hunk 2]
```diff
@@ -195,13 +202,34 @@ pub async fn emit_validator_history_metrics(
...
+
+    let get_vote_accounts_count = get_vote_accounts.len() as i64;
+
+    let vote_program_id = get_vote_program_id();
+    let alive_history_vote_account_count = get_multiple_accounts_batched(&all_history_vote_accounts, client)
+        .await
+        .expect("Cannot fetch all accounts")
+        .iter()
+        .filter_map(|account| account.as_ref()) // Filter out None and unwrap the Some values.
+        .filter(|account| vote_program_id == account.owner) // Filter accounts whose owner matches the vote_program_id.
```
Comment (ebatsell) [inline keepers/validator-keeper/src/lib.rs:217]: should be able to combine both filters into one

**Review by ebatsell — COMMENTED** [review]
Take a look at the lint output as well, the udeps seems to be a different issue

File: keepers/validator-keeper/src/lib.rs (hunk 2 above)
Comment (coachchucksol) [reply keepers/validator-keeper/src/lib.rs:217]: Done

File: keepers/validator-keeper/src/lib.rs (hunk 1 above)
Comment (coachchucksol) [reply keepers/validator-keeper/src/lib.rs:247]: Done

File: keepers/validator-keeper/src/lib.rs [hunk 3]
```diff
@@ -196,15 +201,41 @@ pub async fn emit_validator_history_metrics(
...
-        .len();
+    let get_vote_accounts = get_vote_accounts_with_retry(client, MIN_VOTE_EPOCHS, None).await?;
+
+    let get_vote_accounts_count = get_vote_accounts.len() as i64;
+
+    let vote_program_id = get_vote_program_id();
+    let live_validator_histories_count =
+        get_multiple_accounts_batched(&all_history_vote_accounts, client)
+            .await
+            .expect("Cannot fetch validator history vote accounts")
```
Comment (ebatsell) [inline keepers/validator-keeper/src/lib.rs:212]: `?` here instead of expect

Comment (CoachChuckFF) [reply keepers/validator-keeper/src/lib.rs:212]: Done

**Review by ebatsell — APPROVED** [review]

**Pull request merged**